api/api/src/main/scala/uk/ac/wellcome/platform/api/models/WorkQuery.scala

```diff
@@ -8,7 +8,7 @@ import com.sksamuel.elastic4s.requests.searches.queries.{
 
 sealed trait WorkQuery {
   val queryString: String
-  def query(): Query
+  def query: Query
```

Comment:
:point_up: I've removed the `()`cos whenever I see a method being called with not arguments I think I assume it is causing a side effect (obvs not a major issue and a bit of a bike shed, just thought I would mention why)